docusaurus/docs/reactnative/guides/message_input_customization.mdx

```diff
@@ -229,4 +232,164 @@ const additionalTextInputProps = useMemo(() => {
 >
   ...
 </Channel>
-```
\ No newline at end of file
+```
+
+## Replacing AttachmentPicker With Native Image Picker
+
+Attachment picker opens up as bottom sheet by default.
+It's easy to replace this default attachment picker with a device's native image picker, as shown in screenshots below.
```

Comment:
In order to replace this default behavior with the device's native image picker, you just need to (quick explanation here).

or

The default behavior can be easily replaced with the device's native image picker.